config/main.py

```diff
@@ -5429,6 +5429,83 @@ def disable_use_link_local_only(ctx, interface_name):
     interface_dict = db.get_table(interface_type)
     set_ipv6_link_local_only_on_interface(db, interface_dict, interface_type, interface_name, "disable")
 
+#
+# 'tx_error_threshold' subgroup ('config tx_error_threshold <interface_name> ...')
+#
+
+@interface.group('tx_error_threshold')
+@click.pass_context
+def tx_error_threshold(ctx):
+    """Set or del threshold of tx error statistics"""
+    pass
+
+#
+# 'set_thresh' subcommand
+#
+
+@tx_error_threshold.command()
+@click.argument('interface_name', metavar='<interface_name>', required=True)
+@click.argument('interface_tx_err_threshold', metavar='<interface_tx_err_threshold>', required=True, type=int)
+@click.pass_context
+def set_thresh(ctx, interface_name, interface_tx_err_threshold):
+    config_db = ctx.obj['config_db']
+
+    """Set threshold of tx error statistics"""
+    if interface_name is None:
+        ctx.fail("'interface_name' is None!")
+
+    if clicommon.get_interface_naming_mode() == "alias":
+        interface_name = interface_alias_to_name(interface_name)
+        if interface_name is None:
+            ctx.fail("'interface_name' is None!")
```

Comment:
CLI is a user interface. This is not a user-friendly error message. Please fix all the messages in the files you modified. To understand the expected format of the message please try to run similar commands on the device or search in the existing code.

Reply:
Reference is taken from: interface startup command (Line 4055), interface shutdown command (Line 4096) and interface speed command (Line 4137) all of which use similar error messages ('interface_name' is None!)
will change the error message to 
'Interface name' {} provided is not valid
instead